landscape/categories.md

```diff
@@ -23,24 +23,43 @@ sub-categories. The remaining text is a description of each sub-category.
   - _Secret Management_
     - Tools that help to manage, encrypt or distribute secrets and other sensitive information.
 
-- **Identity & Access Control**
-  - _Identity_
-    - SPIFFE, identity providers, OpenID, LDAP, Okta
-  - _Access Controls_
+- **Identity Lifecycle and & Access Management**
```

Comment:
```suggestion
- **Identity Lifecycle & Access Management**
```

also, I'm not sure how to read this. is it
- identity (lifecycle & access) management, or
- (identity lifecycle) & (access management), or
- ??

I _think_ you mean the second option. is this a tell that maybe `identity lifecycle management` should be its own category, and `access management` its own as well?

Reply:
Would "Authorization" be a reasonable replacement for "Access Management" ?

Reply:
I think authorization seems more narrow to me than access management, but I'm not sure why